restli-tools/src/main/java/com/linkedin/restli/tools/data/ExtensionSchemaValidation.java

```diff
@@ -0,0 +1,150 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.linkedin.restli.tools.data;
+
+import com.linkedin.data.schema.DataSchema;
+import com.linkedin.data.schema.DataSchemaResolver;
+import com.linkedin.data.schema.NamedDataSchema;
+import com.linkedin.data.schema.RecordDataSchema;
+import com.linkedin.data.schema.grammar.PdlSchemaParser;
+import com.linkedin.data.schema.resolver.MultiFormatDataSchemaResolver;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.GnuParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class ExtensionSchemaValidation
```

Comment:
Rename this to end with "cmdLineApp" to follow convention.

Reply:
Follow what convention? All the classes under com/linkedin/restli/tools/data are not end with "cmdLineApp".

Reply:
I believe that is not correct due to historical reasons. We should from now on call all newly created file with "cmdLineApp" because they can be used as command line app:

```
    getProject().javaexec(javaExecSpec -> {
       javaExecSpec.setMain("com.linkedin.restli.tools.data.ExtensionSchemaValidation");
       javaExecSpec.setClasspath(_pathedClasspath);
       javaExecSpec.args(resolverPathStr);
       javaExecSpec.args(_inputDir.getAbsolutePath());
     });
```

Reply:
Updated the class name to "ExtensionSchemaValidationCmdLineApp"